staging/src/k8s.io/api/resource/v1beta1/types.go

```diff
@@ -188,6 +188,18 @@ type ResourcePool struct {
 const ResourceSliceMaxSharedCapacity = 128
 const ResourceSliceMaxDevices = 128
 const PoolNameMaxLength = validation.DNS1123SubdomainMaxLength // Same as for a single node name.
+const BindingConditionsMaxSize = 4
+const BindingFailureConditionsMaxSize = 4
+
+const (
+	// IsPrepared indicates the device ready state.
+	// If NeedToPreparing is True and IsPrepared is True, the scheduler proceeds to Bind.
+	IsPrepared = "dra.example.com/is-prepared"
```

Comment:
This was just an example in the KEP. It doesn't belong into the upstream API. Same for `PreparingFailed`.